x-pack/plugins/task_manager/server/queries/mark_available_tasks_as_claimed.ts

```diff
@@ -128,7 +128,9 @@ function getSortByPriority(definitions: TaskTypeDictionary): estypes.SortCombina
         // TODO: we could do this locally as well, but they may starve
         source: `
           String taskType = doc['task.taskType'].value;
-          if (params.priority_map.containsKey(taskType)) {
+          if (doc['task.priority'].size() != 0) {
+            return doc['task.priority'].value;
+          } else if (params.priority_map.containsKey(taskType)) {
```

Comment:
I was looking at the values for TaskPriority

```
export enum TaskPriority {
  Low = 1,
  Normal = 50,
}
```

Before we start persisting them in the implementation PR, I wonder if we should review the values / order. I think 1 and 50 are ok, but I wonder if 1 should represent top priority and 50 being lower.